File: Java/Driver.java
class Driver extends Account {
   
    public Driver(String name, String document, String email, String password) {
        super(name, document, email, password);

}

    void printDataDriver(){
        System.out.println("Name: " + name);
        System.out.println("Document: " + document);
        System.out.println("Email: " + email);
        System.out.println("Password: " + password);
    }
}

File: JS/Driver.js
class Driver extends Account {
    constructor(name, document, email, password) {
            super(name, document, email, password)

    }

    printDataDriver(){
        console.log("Nombre: " + this.name);
        console.log("Document: " + this.document);
        console.log("Email: " + this.email);
        console.log("Password: " + this.password);

    }

}